charts/ingress-configuration/templates/ipAddrPool.yaml

```
@@ -0,0 +1,23 @@
+{{- $environment := include "apc-global-overrides.environment" . }}
+{{- $metallbNamespace := include "apc-global-overrides.metallbNamespace" . }}
+{{- $clusterName := include "apc-global-overrides.require-clusterName" . }}
+{{- if ne (include "apc-global-overrides.clusterIsHub" .) "true" }}
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: ingress-reserved
+  namespace: {{ $metallbNamespace }}
+  labels:
+    {{- include "ingress-configuration.labels" . | nindent 4 }}
+spec:
+  addresses:
+  - {{ get .Values.ipAddrPool.spoke $clusterName }}
```

Please make this KISS <https://github.com/gr8it/conf-socpoist/pull/348/files#r2497977455>

```suggestion
  - {{ .Values.ipAddrPool }}
```